monai/transforms/spatial/dictionary.py

```diff
@@ -2368,7 +2376,7 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> List[Dict]:
                 new_dict[k] = deepcopy(d[k])
             # fill additional metadata
             new_dict["original_spatial_shape"] = original_spatial_shape
-            new_dict["slices"] = patch[0][1]  # use the coordinate of the first item
+            new_dict["patch_location"] = patch[0][1]  # use the starting coordinate of the first item
```

Comment:
I feel maybe we should also define all these keys of `new_dict` in some Enum to make them easier to use and maintain? I see you already defined something similar, like `WSIPatchKeys`.
CC @ericspod 

Thanks.

Reply:
Sure, I already have them.

Reply:
I updated it with `WSIPatchKeys`.